### ISSUE TITLE
Spins up browser-sync when building

### ISSUE THREAD

**Issue by ccoenen** [Globegitter/broccoli-browser-sync#3]
Whenever i build for production, I see this line in my commandline:

```
> broccoli build out-test
[BS] Reloading Browsers...
```

This leads me to believe that an instance of browser-sync is spinning up even if i'm not going to use it.

**Comment by Globegitter** [comment]
Yeah the plugin is not a 100% ideal. I have written it quite quickly, so it does the job but might have a few unnecessary reloads etc. Can you show me your Brocfile though?

**Comment by ccoenen** [comment]
I'm still in heavy tinkering mode, but in the end i do this:

```
// 80 lines of stuff
// ...
var allTrees = [staticFiles, appJs, appCss, appNpm, appLib];
var bs = browsersync(
  allTrees,
  {
    browserSync: {
      open: false
    }
  }
);

module.exports = mergeTrees(allTrees.concat(bs));
```

**Comment by Globegitter** [comment]
So the plugin initialises browser-sync immediately in the constructer. This could be done of course only when it detects a change, which I did think about but just decided for the easier way. Then also it depends on the trees you put in how quickly it detects a change, etc but I think the first thing is more the issue you are describing.
